onnxruntime/core/providers/webnn/builders/impl/normalization_op_builder.cc

```diff
@@ -214,7 +158,6 @@ void CreateNormalizationOpBuilder(const std::string& op_type, OpBuilderRegistrat
   constexpr static std::string_view op_types[] =
       {
           "BatchNormalization",
-          "GroupNormalization",
```

Comment:
Too bad we lost grouped channel normalization, as it's used in the Olive optimized version of Stable Diffusion unet. I think I'll need to repropose `meanVarianceNormalization` to WebNN to generically handle these, because we don't want to have to introduce more and more subtly different flavors of normalization that vary mainly by axes. I see that having a more direct mappings from ONNX to WebNN did reduce the code length (and red is a nice color), but I suspect (after seeing some helpers in the Chromium codebase added by Shiyi) that we could have written some helpers in ORT to shorten it (it's gone now though anyway 😉). \<resolve me after reading\>